src/app/(auth)/(navigationsBarLayout)/mypage/SideNavigation.tsx

```
@@ -1,50 +1,73 @@
+                onClick={() => setTab(item.id)}
+                className={`w-full flex items-center gap-3 px-4 py-3 rounded-lg text-left transition-all duration-200 ${
+                  tab === item.id
+                    ? 'text-white shadow-lg'
+                    : 'text-gray-400 hover:text-white hover:bg-white/8'
+                }`}
+                style={{
+                  backgroundColor: tab === item.id ? '#214d35' : 'transparent',
+                }}
+              >
+                <Icon size={20} />
+                <span className="font-medium">{item.label}</span>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)